internal/transform/operation_test.go

```diff
@@ -632,6 +633,7 @@ func makeOperationTestOutputs() (transformedOperations []OperationOutput) {
 				"selling_asset_code":   "USDT",
 				"selling_asset_type":   "credit_alphanum4",
 				"selling_asset_issuer": hardCodedDestAccountAddress,
+				"selling_asset_id":     int64(-8205667356306085451),
 				"buying_asset_type":    "native",
```

Comment:
Does `XLM` not get an id assigned? I would expect that even `XLM` should have an id.

Reply:
You're right. I'll add this right now.